docs/CONFIGURING_TASKPROV.md

```diff
@@ -0,0 +1,144 @@
+- Global HPKE Keypair: A non-task-specific HPKE configuration advertised on
+  `/hpke_config` which is used by Taskprov clients to encrypt their reports.
+- Peer Aggregator: Another aggregator operated by a partner organization that
+  supports the Taskprov extension.
+
+Once these entities are present in the database, we can enable Taskprov in
+the aggregator configuration like so:
+```yaml
+taskprov_config:
+  enabled: true
+```
+
+Once this flag is enabled, it must not be disabled. You cannot go from a Taskprov
+aggregator back to a non-Taskprov aggregator.
+
+### Peer Aggregators
+
+Peer aggregators are identified by their HTTP endpoint (as shared in task
+advertisement) and their role relative to our aggregator. For instance, if we
+operate a helper, then the peer aggregator should be a leader.
+
+It is possible for multiple peer aggregators to be configured, and for the the
+same peer endpoint to be configured in both Leader and Helper roles (although
+take note of Janus support for operating as a Leader).
+
+#### Shared Secrets and Parameters
+
+We must agree upon and share the following parameters with peer aggregators 
+out-of-band:
+
+Sensitive values:
+- `verify_key_init`: A random 32-byte string used for deriving task-specific VDAF
+  verify keys.
+- `aggregator_auth_tokens`: A list of bearer tokens used by leader peer aggregator
+  to authenticate to a helper peer aggregator.
+- `collector_auth_tokens`: Unused in Janus, since it cannot operate as a Taskprov
+  leader.
+
+Non-sensitive values:
+- `collector_hpke_config`: The single HPKE configuration of a collector that
+  aggregate shares will be encrypted to. This key belongs to whoever will be
+  collecting on taskprov tasks.
+- `tolerable_clock_skew`: This isn't used in Janus since it can't operate as a
+  Taskprov leader, but is still required for task definitions. It can be set to
+  something arbitrary--the peer does not need to agree upon this value.
+- `report_expiry_age`: How long in seconds to persist client reports. Omit to
+  set no report expiration.
+
+#### Provisioning
+
+Peer aggregators are configured by system operators through the Janus aggregator
+API.
+
+Example for creating a new peer aggregator in the Leader role:
+```bash
+AGGREGATOR_URL=http://localhost:8081
+AGGREGATOR_API_TOKEN="BASE64URL UNPADDED TOKEN HERE"
+
+# Generate shared parameters. Share these with the peer, or ask for the values
+# that they have derived. Notice that these values are base64url encoded with
+# no padding.
+VERIFY_KEY_INIT=$(openssl rand 32 | basenc -w0 --base64url | sed 's/=//g')
+AGGREGATOR_AUTH_TOKEN=$(echo "some_helpful_identifier_$(openssl rand -hex 32)" | 
+    basenc -w0 --base64url | sed 's/=//g')
+
+curl -v \
+    -H "Authorization: Bearer $AGGREGATOR_API_TOKEN" \
+    -H "Accept: application/vnd.janus.aggregator+json;version=0.1" \
+    -H "Content-Type: application/vnd.janus.aggregator+json;version=0.1" \
+    "$AGGREGATOR_URL/taskprov/peer_aggregators" \
+    --data "{
```

Comment:
nit: I think if you used `'` to delimit the value of `--data`, then you wouldn't need to escape `"`s:
```
--data '{ "endpoint": "thing" }'
```

Reply:
Indeed, but `'` will also stop variable substitution `$VAR` due to the $ being literal. I could just avoid substitution, but my hope was to make this command as copy-pastable as possible.